tests/e2e/framework/testInfo.go

```diff
@@ -150,7 +150,8 @@ func (t testInfo) FetchAndSaveClusterLogs(namespace string) error {
 					glog.Warningf("Error during closing file: %v\n", err)
 				}
 			}()
-			dump, err := util.ShellMuteOutput(fmt.Sprintf("kubectl logs %s -n %s", pod, namespace))
+			dump, err := util.ShellMuteOutput(
```

Comment:
The commit message needs to be updated to 

```
Specify container when running kubectl logs
```

Should be `container` but not `cluster`

Reply:
Yes I notice that typo assume as I created this PR and correct it. Ditto for the PR title.